google/gax/api_callable.py

```diff
@@ -155,6 +156,107 @@ def inner(request):
     return inner
 
 
+def _construct_bundling(method_config, method_bundling_override):
+    """Helper for ``construct_settings()``."""
+    if ('bundle_options' in method_config and
+            'bundle_descriptor' in method_config):
+
+        if method_bundling_override == OPTION_INHERIT:
+            bundler = bundling.Executor(BundleOptions(
+                **method_config['bundle_options']))
+        else:
+            if method_bundling_override:
+                bundler = bundling.Executor(method_bundling_override)
+            else:
+                bundler = None
```

Comment:
I think this would be easier to read if instead of nested if else block, here you used elif